packages/forms/components/TextInput.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { inputAppearances, inputContainer, inputValidation } from "../style";
+import {
+  display,
+  flex,
+  flexItem,
+  flush,
+  inputReset,
+  margin,
+  padding,
+  textWeight
+} from "../../shared/styles/styleUtils";
+
+export enum TextInputAppearance {
+  Standard = "standard",
+  Error = "error",
+  Success = "success"
+}
+
+export interface TextInputProps extends React.HTMLProps<HTMLInputElement> {
+  id: string;
+  type: "text" | "number" | "search" | "email" | "password" | "tel" | "url";
+  appearance: TextInputAppearance;
+  inputLabel: string | React.ReactNode;
+  showInputLabel: boolean;
+  validationContent?: string | React.ReactNode;
```

Comment:
I don't love this prop name, but `validationMessage` and `errorMessage` didn't seem right to me either. But only slightly. Open to renaming this anything better.

Reply:
I'm comfortable with this name 👍